tests/Microsoft.Identity.Test.Common/Core/Mocks/MockHttpManager.cs

```diff
@@ -28,31 +28,20 @@ internal sealed class MockHttpManager : IHttpManager,
 
         private readonly IHttpManager _httpManager;
 
-        public MockHttpManager(string testName = null,
-            bool isManagedIdentity = false,
-            Func<MockHttpMessageHandler> messageHandlerFunc = null,
-            Func<HttpClient> validateServerCertificateCallback = null,
-            bool invokeNonMtlsHttpManagerFactory = false) :
-            this(true, testName, isManagedIdentity, messageHandlerFunc, invokeNonMtlsHttpManagerFactory)
-        {
-        }
-
         public MockHttpManager(
-            bool retry,
+            bool disableInternalRetries = false,
```

Comment:
Let's not add the default value here and rely on what is passed

Reply:
resolved offline